### DISCUSSION TITLE
Melhora da pontuação de qualidade com MercadoPago.JS V2

### DISCUSSION THREAD

**Discussion by andoraul** [mercadopago/sdk-react#89] [General]
No teste de Qualidade da Integração estamos perdendo 10 pontos por conta do SDK de front end. 
A mensagem é a seguinte: 

"Instale o SDK MercadoPago.JS V2 para simplificar o uso e interagir de forma segura com nossas APIs."

Existe alguma maneira de corrigir isso? 
Estou usando o sdk-react 0.0.16.
Agradeço desde já!

**Comment by meliguicarvalho** [comment]
Olá Raul, bom dia!

Obrigado por entrar em contato, voce pode por favor providenciar mais detalhes sobre sua integração, qual produto esta tentando integrar com nossa SDK, e que testes esta rodando para receber essa mensagem?
Isso vai nos permitir fazer uma análise melhor da situação.

Obrigado!

**Comment by andoraul** [comment]
Bom dia! Obrigado pela resposta. 

Estou utilizando o Payment e o StatusScreen. Os testes estão ocorrendo dentro do esperado, nenhuma mensagem de erro nem avisos são exibidos. Portanto, passamos para o ambiente de produção. 

No portal do Mercado Pago Developers existe o teste de Qualidade da integração. Aquele onde colocamos o ID do pagamento, ele calcula a nota, e retorna um check list de recomendações para melhorar a qualidade da integração. Estamos em busca de otimizá-la ao máximo para evitar compras recusadas que estão simplesmente sabotando nosso negócio. 

Então, um dos pontos dessa lista é justamente a recomendação citada. 

"Instale o SDK MercadoPago.JS V2 para simplificar o uso e interagir de forma segura com nossas APIs."

No entanto, notei que no arquivo @mercadopago/sdk-js/dist/index.js existe a seguinte linha:

const SDK_MERCADOPAGO_URL = 'https://sdk.mercadopago.com/js/v2';

Por isso questionei se existe algo que possamos fazer para evitar que esse falso positivo prejudique nossa nota.

**Reply by meliguicarvalho** [reply]
Perfeito, obrigado pelos detalhes!

Vamos investigar e assim que tenha mais detalhes te informo.

**Reply by meliguicarvalho** [reply]
Raul, estivemos analisando internamente a questão que você levantou, e, para avançarmos mais nas investigações, vamos precisar de mais detalhes.

Por isso preciso te pedir para abrir um ticket de suporte.

Para abrir o ticket, acesse, a partir de sua conta de desenvolvedor, essa [página](https://www.mercadopago.com.br/developers/pt/support/center/tickets).

No ticket por favor inclua a explicação que você me deu, além do número do pagamento que está gerando o erro, o número da sua aplicação, e o seu User ID. Essas informações pode encontrar na sua [página de desenvolvedor](https://www.mercadopago.com.br/developers/panel/app), navegando para a aplicação que está resultando nesse problema.

Com o ticket aberto, conseguimos dar seguimento internamente de forma segura, sem compartilhar essas informações sensíveis da sua conta abertamente. Precisamos da abertura do ticket somente para preservar a segurança e integridade da comunicação.

Obrigado!

**Reply by andoraul** [reply]
Certo, vou abrir o ticket. Obrigado pela atenção!

**Reply by wfern** [reply]
@andoraul @meliguicarvalho Caso tenha sido problema no seu lado da integração e não envolver informações sensíveis posta a solução aqui para ajudar outras pessoas. Tenho visto que esse está sendo um problema bastante comum. Obrigado!

Também já abri um ticket, no meu caso tentei com e sem `npm` e também com o checkout transparent e checkout bricks mas sem sucesso em nenhum dos casos.

**Comment by duarte-dot** [comment]
conseguiu resolver amigo? to sofrendo o msm problema kkkk

**Reply by felipeov1** [reply]
Cara eu também, ta foda. Estou com problemas no sdk e nos certificados sendo que ta tudo certo.

**Comment by willcansilv-gif** [comment]
Olá pessoal, estou enfrentando um problema crítico na aplicação.

Minha integração é via WooCommerce, modelo Marketplace, e já implementei o SDK V2 e o script de segurança conforme a documentação:

SDK V2 carregado com Public Key.

Script security.js com view="checkout" e output="device_id" inserido no cabeçalho do checkout.

SSL e TLS 1.2 validados.

O Problema: Mesmo com as transações ocorrendo, a pontuação de qualidade permanece em 75 pontos. O item "Formulário de Cartões - PCI Compliance" aparece como pendente, mas ao clicar no botão, a interface do painel não carrega o formulário ou retorna erro de página não encontrada.

Além disso, como a pontuação não atinge o nível exigido, o Split de Pagamentos (Marketplace) está sendo ignorado pela API, e o valor cai integralmente na conta do integrador, apesar de enviarmos corretamente o marketplace_seller_id e o application_fee.

Alguém conseguiu resolver essa trava de pontuação ou existe algum bypass via suporte para contas de Marketplace?
